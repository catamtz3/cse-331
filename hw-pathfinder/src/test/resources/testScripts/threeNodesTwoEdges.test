CreateGraph twoNodes

AddNode twoNodes a
AddNode twoNodes b
AddNode twoNodes c

AddEdge twoNodes a b 10.0
AddEdge twoNodes b c 5.0

ListNodes twoNodes

ListChildren twoNodes a

FindPath twoNodes a c