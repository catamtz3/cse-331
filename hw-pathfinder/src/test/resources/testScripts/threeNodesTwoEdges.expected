created graph twoNodes

added node a to twoNodes
added node b to twoNodes
added node c to twoNodes

added edge 10.0 from a to b to twoNodes
added edge 5.0 from a to c to twoNodes

twoNodes contains: a, b, c

the children of a in twoNodes are: b(10.0) c(5.0)

path from a to c:
a to b with weight 10.0
b to c with weight 5.0
total cost: 15.0